src/lib/api.js

```diff
@@ -377,14 +379,18 @@ const searchItems = async function (collectionId, queryParameters, backend, endp
   const ids = extractIds(queryParameters)
   const collections = extractCollectionIds(queryParameters)
 
+  const collectionsThatExist = Array.isArray(collections)
```

Comment:
I would prefer if we not have to make the calls to check if the indicies exist first before calling the search. I was thinking the fix would be to add allow_no_indices=true to the call to client.search in es.js.